src/app/pages/storage/modules/disks/components/manual-test-dialog/manual-test-dialog.component.ts

```diff
@@ -99,6 +100,10 @@ export class ManualTestDialogComponent {
             next: (result) => {
               if (result?.progress) {
                 this.progressTotalPercent = result.progress.percent / 100;
+
+                if (this.progressTotalPercent === 100) {
+                  this.dialogRef.close();
```

Comment:
This doesn't look pretty. Wouldn't the job complete and dialog be closed on its own?

Reply:
> This doesn't look pretty. Wouldn't the job complete and dialog be closed on its own?

I Just added extra check to close it once progress is 100%

Reply:
Is it necessary?

Reply:
> Is it necessary?

nope, at all

Reply:
I'll remove it.